plugins/jobs/web_client/views/JobGraphWidget.js

```diff
@@ -110,13 +108,11 @@ const JobGraphWidget = View.extend({
             config.height = this.$('.g-jobs-graph').height();
             config.data[0].values = vegaData;
             config.scales[1].type = this.yScale;
-            config.scales[2].domain = jobs.map(job => job.get('_id'));
-            config.scales[2].range = jobs.map(job => {
-                let datetime = moment(job.get('updated')).format('MM/DD');
-                return datetime;
-            });
-            config.scales[3].domain = jobs.map(job => job.get('_id'));
-            config.scales[3].range = jobs.map(job => job.get('title'));
+            config.scales[2].domain = this.collection.pluck('_id');
+            config.scales[2].range = this.collection.map(
+                (job) => moment(job.get('updated')).format('MM/DD'));
+            config.scales[3].domain = this.collection.pluck('_id');
```

Comment:
I personally would prefer standard methods instead of backbone/underscore particular way, because I think standard stuff is easier to read for anyone. I also think using a local variable **jobs** looks better than **this.collection** and may has a little better performance. 
But this works fine.